app/js/arethusa.core/factories/tree.js

```diff
@@ -739,12 +754,12 @@ angular.module('arethusa.core').factory('Tree', [
         applyViewMode();
       });
 
-     
+
       // if a tree was rendered before it is visible
       // refreshing will rerender it and fix display bugs
       navigator.onRefresh(function() {
         render();
-        scope.perfectWidth();
+        scope.refreshWidth();
```

Comment:
Is there are reason to have separate `perfectWidth` and `refreshWidth` functions? Would it be possible to replace the logic in `perfectWidth` with the logic now in `refreshWidth` or would that cause breaking changes?

Reply:
I'm not really sure.  The perfectWidth function is perhaps misnamed (given that perfection is subjective...). What it does is zoom the display to make the diagram take up as much of the width as it can, regardless of how wide the tree is.  I was thinking it was safer to just make a new function in case anyone really likes the perfectWidth as it is. It's really more of a zoomWidth than perfectWidth I think.

Reply:
That makes sense. Thanks for explaining the reasoning.